ci/terraform/oidc/shared.tf

```diff
@@ -59,8 +58,8 @@ locals {
   pepper_ssm_parameter_policy                         = data.terraform_remote_state.shared.outputs.pepper_ssm_parameter_policy
   lambda_code_signing_configuration_arn               = data.terraform_remote_state.shared.outputs.lambda_code_signing_configuration_arn
   auth_code_store_signing_configuration_arn           = data.terraform_remote_state.shared.outputs.auth_code_store_signing_configuration_arn
-  di_auth_ext_api_id                                  = var.support_auth_orch_split ? data.terraform_remote_state.auth-ext-api[0].outputs.di_auth_ext_api_id : ""
-  vpce_id                                             = var.support_auth_orch_split ? data.terraform_remote_state.auth-ext-api[0].outputs.vpce_id : ""
+  di_auth_ext_api_id                                  = data.terraform_remote_state.auth-ext-api.outputs.di_auth_ext_api_id
```

Comment:
why is this no longer `auth-ext-api[0]`?

Reply:
I removed the count argument on line 18 as that was also conditional on support_auth_orch_split.